### PR TITLE
Add Dockerfiles for Ubuntu 22.04 (Jammy)

### PR DESCRIPTION
**Pull request by mthalman**
This starts with a cherry-pick of the commit which added these Dockerfiles in the nightly branch. But I've added a separate commit to remove the Arm32 Dockerfiles due to https://github.com/dotnet/runtime/issues/66310. Once that fix is available in the next servicing release, we can just revert this Arm32 commit.

Related to https://github.com/dotnet/dotnet-docker/issues/3350

### CODE REVIEW

File: README.runtime-deps.md [hunk 1]
```diff
@@ -108,7 +110,6 @@ Tags | Dockerfile | OS Version
 -----------| -------------| -------------
 7.0.0-preview.3-bullseye-slim-arm32v7, 7.0-bullseye-slim-arm32v7, 7.0.0-preview.3, 7.0.0-preview.3-bullseye-slim, 7.0, 7.0-bullseye-slim | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/3.1/bullseye-slim/arm32v7/Dockerfile) | Debian 11
 7.0.0-preview.3-alpine3.15-arm32v7, 7.0-alpine3.15-arm32v7, 7.0-alpine-arm32v7, 7.0.0-preview.3-alpine3.15, 7.0-alpine3.15, 7.0-alpine | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/5.0/alpine3.15/arm32v7/Dockerfile) | Alpine 3.15
-7.0.0-preview.3-focal-arm32v7, 7.0-focal-arm32v7, 7.0.0-preview.3-focal, 7.0-focal | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/3.1/focal/arm32v7/Dockerfile) | Ubuntu 20.04
```
Comment (MichaelSimons) [inline README.runtime-deps.md:111]: It feels a bit weird to drop all Ubuntu images.  Should this one be preserved until a jammy image is available?

Comment (mthalman) [reply README.runtime-deps.md:111]: I've rebased the last commit to switch from Jammy to Focal for 7.0.

File: README.runtime-deps.md [hunk 2]
```diff
@@ -108,7 +110,7 @@ Tags | Dockerfile | OS Version
...
+7.0.0-preview.3-focal-arm32v7, 7.0-focal-arm32v7, 7.0.0-preview.3-jammy, 7.0-jammy | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/3.1/focal/arm32v7/Dockerfile) | Ubuntu 20.04
```
[the remark this answers is not in the document]
Comment (mthalman) [reply README.runtime-deps.md]: I've fixed this. Please see the latest commit. If it's good, I'll squash it with its parent.

**Review by MichaelSimons — APPROVED** [review]
LGTM

**Pull request merged**